engine/engine.go

```diff
@@ -13,19 +13,24 @@ import (
 	"golang.org/x/crypto/openpgp/clearsign"
 
 	"github.com/cenkalti/backoff"
+	"github.com/pkg/errors"
 
 	"github.com/ONSdigital/dp-deployer/config"
 	"github.com/ONSdigital/dp-deployer/ssqs"
 	"github.com/ONSdigital/dp-net/request"
-	"github.com/ONSdigital/goamz/aws"
-	"github.com/ONSdigital/goamz/sqs"
 	"github.com/ONSdigital/log.go/v2/log"
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+
 )
 
 // maxConcurrentHandlers limit on goroutines (each handling a message)
 const maxConcurrentHandlers = 50
 
-var sendMessage func(string) error
+var sendMessage func(context.Context, string) error
+
+var loadDefaultConfigFunc func(context.Context, ...func(*awsconfig.LoadOptions) error) (aws.Config, error)=  awsconfig.LoadDefaultConfig
```

Comment:
the lack of a space before the `=` in the above
shows that you've not run `go fmt` on these files, 
I think we strongly recommend ([if not mandate](https://github.com/ONSdigital/dp/blob/main/training/culture-and-process/PR_REVIEW_AND_RESPONSE_GUIDANCE.md#go)) that `go fmt` is run automagically (on 'save') in your editor
(I've just run `go fmt ./...` on the repo and it changed quite a few lines)